### DISCUSSION TITLE
Small bug? i found

### DISCUSSION THREAD

**Discussion by Trioplane** [community/community#15025] [Copilot Conversations]
When pressing CTRL+ENTER while github copilot is activated, it will show this
![image](https://user-images.githubusercontent.com/87922281/163662587-53798010-f485-4d10-bf0e-adf942eefa2d.png)
(Ignore the error)
But when copilot is disabled it will still be able to do that

![image](https://user-images.githubusercontent.com/87922281/163662623-da97fb79-c025-40cb-a0d8-f1f53deb57b8.png)
also for some reason the github copilot logo is not visible for me

Steps:
Disable copilot by clicking it on the bottom right, press CTRL+ENTER

Expected Behavior:
CTRL+Enter will not work. (IF this is intended then this is useless)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
